src/resources/postcss/components/_search.pcss

```diff
@@ -68,26 +69,82 @@
 			}
 
 			&:first-child {
-				flex: 1 1 34.5%;
+				flex: 1 1 100%;
 			}
 
 			&:nth-child(2) {
-				flex: 1 1 37.5%;
+				flex: 1 1 100%;
 			}
 
-			&:nth-child(3) {
-				flex: 1 1 28%;
+			&:last-child {
+				.tribe-common-c-search__input {
+					@media (--viewport-medium) {
+						border-right: 0;
+					}
+				}
+
 			}
+
 		}
 	}
 
 	.tribe-common-c-search__input {
+
 		@media (--viewport-medium) {
-			border: none;
 			border-right: 1px solid var(--color-border-default);
 			height: auto;
 			padding: 7px 18px 8px;
 		}
+
+		&:focus {
+			outline: none;
+		}
+	}
+
+	.tribe-common-c-search__input--keyword {
+		background-image: svg-inline(search);
+		background-repeat: no-repeat;
+		background-size: 16px;
+		background-position: right center;
+
+		@media (--viewport-medium) {
+			border: none;
+			border-right: 1px solid var(--color-border-default);
+			height: auto;
+			padding: 7px 18px 7px 40px;
```

Comment:
We should not be overriding padding styles from form control styles, otherwise there's no reason for us to have them. I see that this is to get the `border-right` style correct here. I'd do a few things here:

1. set that `40px` padding in the form control styles so default styles are applied with that padding
2. move the `border-right` style to the wrapper div (or even better, move it to `.tribe-common-c-search__input-group > *`)
3. to get the correct border height, apply a negative margin on top and bottom on `.tribe-common-c-search__input`.

This way, we get the full height for the input, but the border is still the smaller height as per the design.

Reply:
Actually, the padding of these overrides are to include the icons on the different inputs.

Reply:
Right, so the padding values in `.tribe-common-form-control-text__input`, I don't see them in the design anywhere. I believe if we only have the event bar styles given to us for input styles, we should use that as the default. We can always come back and shuffle around what's common styles and what's specific to events bar when a new input style comes, but until then, having multiple different padding values is unnecessary CSS and can be confusing when we do try to make an update.

Reply:
Right, the padding is not anymore in the design anymore because the two input fields we have, are now using icons. I thought keeping the default values for the default input, and adding modifiers, would allow us to add a new field without any trouble, and that field would just look ok and without any strange padding.

It's like -> Default field has a default padding which is OK for text. Modifiers have their own padding because they have an icon and they need space. If not the text will be on top of the icon. If you want I can make a modifier for icon. I feel like that would be OK, but at the same time it'd be better to keep that on the input rather than the wrapper.

Reply:
I hear you, let's chat about this and figure out the way forward.